adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -302,6 +342,33 @@ public String getIdToken() {
         return mIdToken;
     }
 
+    /**
+     * Gets if the returned token is valid in terms of extended lifetime
+     *
+     * @return True if the returned token is valid in terms of extended lifetime
+     */
+    final public boolean isExtendedLifeTimeToken() {
+        return mIsExtendedLifeTimeToken;
+    }
+
+    /**
+     * Sets the property isExtendedLifeTimeToken to indicate whether the token
+     * being returned is a token only valid in terms of extended lifetime
+     *
+     * @param isExtendedLifeTimeToken
+     */
+    final void setIsExtendedLifeTimeToken(final boolean isExtendedLifeTimeToken) {
+        this.mIsExtendedLifeTimeToken = isExtendedLifeTimeToken;
```

Comment:
I like what she has currently.

---
In reply to: [70552948](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r70552948) [](ancestors = 70552948)